SimG4Core/MagneticField/src/FieldStepper.cc

```diff
@@ -53,7 +56,9 @@ void FieldStepper::selectStepper(const std::string &ss) {
   else if (ss == "G4DormandPrince745")
     theStepper = new G4DormandPrince745(theEquation);
   else if (ss == "G4TDormandPrince45")
```

Comment:
@VinInn , old DormandPrince better do not remove, living the possibility to compare CPU performance and results. G4TDormandPrince45CMS can be added as an alternative: we have more than 10 steppers, some we did not used at so far. Addition of an extra changes nothing.

Reply:
ok.
where I can change the default to "G4TDormandPrince45CMS"?

Reply:
Inside SimG4Core/Application/python/g4SimHits_cfi.py